src/Util/Tokens.php

```diff
@@ -154,6 +154,10 @@
     define('T_ATTRIBUTE', 'PHPCS_T_ATTRIBUTE');
 }
 
+if (defined('T_READONLY') === false) {
+    define('T_READONLY', 'PHPCS_T_READONLY');
+}
+
```

Comment:
This needs to be moved down to below the `// Some PHP 8.1 tokens, replicated for lower versions.` marker.

Reply:
Fixed.